### DISCUSSION TITLE
Mipmapping and Anisotropic Filtering Features in Viewport

### DISCUSSION THREAD

**Discussion by MotivaCG** [libigl/libigl#2397] [General]
I am currently working with libigl and have noticed that mipmapping and anisotropic filtering for textures are not available in the Viewer. These features are crucial for enhancing the visual quality of textures, especially when viewed at oblique angles or from a distance.

This is particularly important for scenes where surface details are crucial for the visual outcome.

Is this something that you could consider in the future?

Thank you.

**Comment by alecjacobson** [comment]
Hi motivacg, I have no plans to add this feature. I might merge a pr that added it.

**Reply by MotivaCG** [reply]
It makes sense :)
